### ISSUE TITLE
Adjustable Modulator amount

### ISSUE THREAD

**Issue by tmhglnd** [tmhglnd/mercury#99]
Have a setting in the preferences to adjust the Modulator amount. Currently there is a limit of 8, but if someone wants more this could be adjusted with the setting.

**Comment by HighHarmonics2** [comment]
I like this idea. Having flexibility to expand the Modulator count would be useful. I'm finding that I often have 3 - 4 modulators for a single instrument - particularly my custom instruments. I'm finding the constraint of 8 to be challenging, forcing me to comment/uncomment a lot.  It would be good to know what the tradeoffs are - more Modulators = higher CPU utilization (?I assume) - but by how much? Are there other reasons for having a default limit on Modulators?

**Comment by tmhglnd** [comment]
Yes, the mean reason is that everything in Max needs to be loaded, even if things are not doing much, since it is all done with voices in poly~ objects. More modulators and instruments means even longer loading time on startup and some additional cpu usage and overhead as well, yes (although not much if they are not in use, but still). I prefer to keep it lightweight, and then if you want to have 8+ modulators/instruments you can adjust the setting. How much more cpu usage it would, i don't know.
